test/remote/gateways/remote_midtrans_test.rb

```diff
@@ -20,6 +37,13 @@ def test_purchase_when_valid_card_then_success
     assert_equal response.params["status_code"], "200"
   end
 
+  def test_purchase_when_valid_card_with_3ds_then_success
+    response = @gateway.purchase(@amount, @accepted_card, @card_with_3ds_payment_options)
+    assert_success response
+    assert_equal response.params["status_code"], "201"

```

Comment:
As per the Rails convention and [assert_equal](https://apidock.com/ruby/Test/Unit/Assertions/assert_equal) API documentation `assert_equal` function prototype is 
```
assert_equal(expected, actual)
```

If we want to stick with this convention, we need to change this to 
```
assert_equal "201", response.params["status_code"]
```

Let me know in case team has any other concerns with this.